repos/system_upgrade/common/actors/checkldsoconf/libraries/checkldsoconfiguration.py

```diff
@@ -0,0 +1,133 @@
+import glob
+import os
+
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import InstalledRedHatSignedRPM
+
+LD_SO_CONF_DIR = '/etc/ld.so.conf.d'
+LD_SO_CONF_MAIN = '/etc/ld.so.conf'
+LD_SO_CONF_DEFAULT_INCLUDE = 'ld.so.conf.d/*.conf'
+LD_SO_CONF_COMMENT_PREFIX = '#'
+LD_LIBRARY_PATH_VAR = 'LD_LIBRARY_PATH'
+FMT_LIST_SEPARATOR_1 = '\n- '
+FMT_LIST_SEPARATOR_2 = '\n    - '
+
+
+def _read_file(file_path):
+    with open(file_path, 'r') as fd:
+        return fd.readlines()
+
+
```

Comment:
@PeterMocary  the actor is in the checksphase - where we allow to work only with data obtained via msgs. any  interaction with the system is prohibited here (r/w files, shell cmds execution - run, etc.). In this case we need an actor that produce msg with all important data, and this actor will consume such a msg and create a report based on it.

Reply:
Separated into two actors.